### PR TITLE
257 search

### LINKED ISSUE #257: change surgery filter term
The surgery filter looks like this now:

image

change it to
•	Surgery: Yes v No

### LINKED ISSUE #258: change seach filters for Nephrectomy type
Right now it looks like this:
image

- [x] Change test date total to Radical
- [x] Change real date total to Radical

### LINKED ISSUE #259: Change search filter for Nephrectomy Robotic Assist
Right now it looks like this:
image

- [x] Change schema: Nephrectomy Robotic Assist is a string instead of boolean

- [x] Change test data to Yes or No.

- [x] Change real data to Yes or No.

### LINKED ISSUE #261: Change search filters for all stage
- [x] for example change pN stage  to pN stage at presentaion. Need changes for pN, pT, pM and TNM stage. 

- [x] Also for specific stage, letter should be lower case. change it for test data

- [x] Also for specific stage, letter should be lower case. change it for real data

### LINKED ISSUE #263: Change search filters for germline mutation
- [x] Too few germline cases. So we will take it off for now.
- [x] IHC should go in brackets. For others include sequencing in brackets. for example: change **IHCMLH1** to **MLH1(IHC)** , change **MAX** to **MAX(Sequencing)**. Both schema and test data need to be updated.
- [x] IHC should go in brackets. For others include sequencing in brackets. for example: change **IHCMLH1** to **MLH1(IHC)** , change **MAX** to **MAX(Sequencing)**. real data need to be updated.

### LINKED ISSUE #266: update about the team
Include Christina and Roy in team. Add Dr. B as Advisor.

### LINKED ISSUE #267: change typo for Metastasis Site and Metastasis Type
Metastasis Site and Metastasis Type

### PR DESCRIPTION
**Pull request by mingjiecn**
close #267 

close #266 
image

close #263 after real data is updated:
image

close #261 after real data is updated:
image

close #259 
image

close #258 after real data is updated:
image

close #257 
image

### CODE REVIEW

**Comment by vsmalladi** [comment]
Updated real data to change Total -> Radical: close #258

**Comment by vsmalladi** [comment]
Updated real data close #261

**Comment by vsmalladi** [comment]
Updated real data close #263

File: src/encoded/schemas/biospecimen.json [hunk 1]
```diff
@@ -209,16 +209,16 @@
             "title": "Genomics File Output Type"
         },
         "surgery.pathology_report.t_stage": {
-            "title": "pT Stage"
+            "title": "pT Stage at Presentaion"
         },
         "surgery.pathology_report.n_stage": {
-            "title": "pN Stage"
+            "title": "pN Stage at Presentaion"
```
Comment (vsmalladi) [inline src/encoded/schemas/biospecimen.json]: @mingjiecn Spelling error should be "Presentation"

File: src/encoded/schemas/biospecimen.json [hunk 2]
```diff
@@ -209,16 +209,16 @@
...
         },
         "surgery.pathology_report.m_stage": {
-            "title": "pM Stage"
+            "title": "pM Stage at Presentaion"
```
Comment (vsmalladi) [inline src/encoded/schemas/biospecimen.json]: @mingjiecn Spelling error should be "Presentation"

File: src/encoded/schemas/biospecimen.json [hunk 3]
```diff
@@ -209,16 +209,16 @@
...
         },
         "surgery.pathology_report.ajcc_tnm_stage": {
-            "title": "TNM Stage"
+            "title": "TNM Stage at Presentaion"
```
Comment (vsmalladi) [inline src/encoded/schemas/biospecimen.json]: @mingjiecn Spelling error should be "Presentation"

File: src/encoded/schemas/patient.json [hunk 4]
```diff
@@ -151,22 +151,22 @@
             "title": "Tumor Histologic Subtypes"
         },
         "surgery.pathology_report.ajcc_p_stage": {
-            "title": "pT Stage"
+            "title": "pT Stage at Presentaion"
```
Comment (vsmalladi) [inline src/encoded/schemas/patient.json]: @mingjiecn Spelling error should be "Presentation"

File: src/encoded/schemas/patient.json [hunk 5]
```diff
@@ -151,22 +151,22 @@
...
         },
         "surgery.pathology_report.n_stage": {
-            "title": "pN Stage"
+            "title": "pN Stage at Presentaion"
```
Comment (vsmalladi) [inline src/encoded/schemas/patient.json]: @mingjiecn Spelling error should be "Presentation"

File: src/encoded/schemas/patient.json [hunk 6]
```diff
@@ -151,22 +151,22 @@
...
         },
         "surgery.pathology_report.m_stage": {
-            "title": "pM Stage"
+            "title": "pM Stage at Presentaion"
```
Comment (vsmalladi) [inline src/encoded/schemas/patient.json]: @mingjiecn Spelling error should be "Presentation"

File: src/encoded/schemas/surgery.json [hunk 7]
```diff
@@ -165,22 +165,22 @@
             "title": "Tumor Size"
         },
         "pathology_report.ajcc_p_stage": {
-            "title": "pT Stage"
+            "title": "pT Stage at Presentaion"
```
Comment (vsmalladi) [inline src/encoded/schemas/surgery.json]: @mingjiecn Spelling error should be "Presentation"

File: src/encoded/schemas/surgery.json [hunk 8]
```diff
@@ -165,22 +165,22 @@
...
         },
         "pathology_report.n_stage": {
-            "title": "pN Stage"
+            "title": "pN Stage at Presentaion"
```
Comment (vsmalladi) [inline src/encoded/schemas/surgery.json]: @mingjiecn Spelling error should be "Presentation"

File: src/encoded/schemas/surgery.json [hunk 9]
```diff
@@ -165,22 +165,22 @@
...
         },
         "pathology_report.m_stage": {
-            "title": "pM Stage"
+            "title": "pM Stage at Presentaion"
```
Comment (vsmalladi) [inline src/encoded/schemas/surgery.json]: @mingjiecn Spelling error should be "Presentation"

File: src/encoded/schemas/surgery.json [hunk 10]
```diff
@@ -165,22 +165,22 @@
...
-            "title": "Metasis Site"
+            "title": "Metastasis Site"
         },
         "pathology_report.metasis_details.type": {
-            "title": "Metasis Type"
+            "title": "Metastasis Type"
         },
         "pathology_report.ajcc_tnm_stage": {
-            "title": "TNM Stage"
+            "title": "TNM Stage at Presentaion"
```
Comment (vsmalladi) [inline src/encoded/schemas/surgery.json]: @mingjiecn Spelling error should be "Presentation"

**Review by vsmalladi — CHANGES_REQUESTED** [review]
Minor typos but everything looks good.

**Comment by mingjiecn** [comment]
corrected the typo

**Review by vsmalladi — APPROVED** [review]

**Pull request merged**